src/main/java/org/stellar/sdk/AllowTrustOperation.java

```diff
@@ -47,7 +47,7 @@ public boolean getAuthorize() {
   }
 
   @Override
-  org.stellar.sdk.xdr.Operation.OperationBody toOperationBody() {
+  org.stellar.sdk.xdr.Operation.OperationBody toOperationBody(AccountConverter accountConverter) {
```

Comment:
Forcing this parameter for operations that don't support muxed accounts is a little awkward to me as an API consumer, but I'm not familiar enough with Java patterns to suggest alternatives.

Reply:
this method is only used internally. any method which does not have a `public` modifier is only visible within the SDK.

I just pushed another commit which ensures that, for all `public` methods that handle account strings, we have two implementations: one which allows you to provide your own account converter and another which uses `AccountConverter.disableMuxed()` by default